pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/relocation/SegmentRelocator.java

```diff
@@ -80,6 +80,7 @@ public class SegmentRelocator extends ControllerPeriodicTask<Void> {
 
   private final Set<String> _waitingTables;
   private final BlockingQueue<String> _waitingQueue;
+  private final Set<String> _tablesUndergoingRebalance;
```

Comment:
nit: let's annotate this with `@Nullable` to reduce the chances of NPEs since this is gonna be `null` for the sequential mode.

Reply:
done